mixins/global.js

```diff
@@ -1,7 +1,25 @@
 // Global mixin so that every component can access the logged in state and user.
 import Vue from 'vue'
+import { REGION_WALES, TAGLINE_LONDON, TAGLINE_WALES } from '@/regions'
```

Comment:
What's the @ here?  Don't think I've seen that before.

Reply:
It's like ~ but avoids bothering with the extension.  See https://stackoverflow.com/a/42753045/775258